docs/DeveloperGuide.adoc

```diff
@@ -581,29 +581,49 @@ Step 1. The user starts up the application with an initial list loaded from exte
 
 :figure-caption!:
 .Example list on startup
-image::update/updateDG/UpdateStep1.png[]
+image::update/updateDG/UpdateStep1.png[,650]
 
 Step 2. The user inputs **update i/1 n/Government Bursary v/600** to update the 1st transaction's name to "Government Bursary" and value to "600". Input is parsed by `UpdateCommandParser#parse` which creates an `**UpdateCommand**`.
 
-NOTE: The 1st transaction specified here is an `**Income**`
+[NOTE]
+The 1st transaction specified here is an `**Income**`
 
 .Example user input for update command
-image::update/updateDG/UpdateStep2.png[]
+image::update/updateDG/UpdateStep2.png[,650]
 
 Step 3. `UpdateCommand#execute` creates a new transaction that reflects the changes and gets the index of current transaction to be updated.
 
 .Creation of new updated transaction in `UpdateCommand#execute()`
-image::update/updateDG/UpdateStep3.png[]
+image::update/updateDG/UpdateStep3.png[,650]
 
 Step 4. `UpdateCommand#execute` replaces original transaction in the list with the updated transaction.
 
-image::update/updateDG/UpdateStep4Part1.png[]
-image::update/updateDG/UpdateStep4Part2.png[]
-.Replacement of original transaction with updated transaction in `UpdateCommand#execute()`
-image::update/updateDG/UpdateStep4Part3.png[]
+image::update/updateDG/UpdateStep4Part1.png[,650]
+image::update/updateDG/UpdateStep4Part2.png[,650]
+.Replacement of original transaction with updated transaction by `UpdateCommand#execute()`
+image::update/updateDG/UpdateStep4Part3.png[,650]
+
+The following code snippet from `UpdateCommand#execute` shows the creation of an updated transaction and the replacement of the original transaction with the updated one:
+
+.UpdateCommand#execute snippet
```

Comment:
Missing `()`, maybe don't need to state "snippet" since the preface is quite clear